frontend/lib/features/audio-player/player_button.widget.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+import 'package:just_audio/just_audio.dart';
+import 'package:mbl/themes/themes.dart';
+
+Widget playerButton(PlayerState? playerState, AudioPlayer player) {
```

Comment:
the player button is just a button and should be dumb. It's either play or pause. The player change should be changed within a callback. Just like the image/grid switch